code-challenges.test.js

```diff
@@ -10,45 +12,105 @@
 
 // b) Create the function that makes the test pass.
-
-
+// Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
+// Use .shift(), to remove the first item from the array
+const arrayMixUpColors = (array) => {
+  array.shift();
+  // Use .sort(), to sort the items within the array in ascending order
+  let newArr = array.sort((a, b) => a.sort - b.sort);
+  // Iterate over array using .map()
+  array.map(({ value }) => value);
+  return newArr;
+};
+// Green - Passed
 
 // --------------------2) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var nums1 = [3, 56, 90, -8, 0, 23, 6]
-// Expected output: [-8, 90]
-var nums2 = [109, 5, 9, 67, 8, 24]
-// Expected output: [5, 109]
-
+describe("minNMax", () => {
+  it("takes an array of numbers and returns an array of number, then returns an array of the minimum and maximum numbers in that order", () => {
+    var nums1 = [3, 56, 90, -8, 0, 23, 6];
+    // Expected output: [-8, 90]
+    var nums2 = [109, 5, 9, 67, 8, 24];
+    // Expected output: [5, 109]
 
+    expect(minNMax(nums1)).toEqual([-8, 90]);
+    expect(minNMax(nums2)).toEqual([5, 109]);
+  });
+});
+// Red - Failed (not defined)
 
 // b) Create the function that makes the test pass.
+// Use Math.min() to get the smallest number and Math.max to get the largest number.
+// Place both numbers into an array.
 
-
+const minNMax = (array) => {
+  return [Math.min(...array), Math.max(...array)];
+};
+// Green - Passed
 
 // --------------------3) Create a function that takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator to pass in a dynamic number of arguments.
 
 // a) Create a test with an expect statement using the variables provided.
 
-var testArray1 = [3, 7, 10, 5, 4, 3, 3]
-var testArray2 = [7, 8, 2, 3, 1, 5, 4]
-// Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
-
+describe("allNumsNoRepeats", () => {
+  it("Create a function that takes in two arrays as arguments and returns one array with no duplicate values.", () => {
+    var arrayOne = [3, 7, 10, 5, 4, 3, 3];
+    var arrayTwo = [7, 8, 2, 3, 1, 5, 4];
+    // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
 
+    expect(allNumsNoRepeats(arrayOne, arrayTwo)).toEqual([
+      3, 7, 10, 5, 4, 8, 2, 1,
+    ]);
+  });
+});
+// Red - Failed (not defined)
 
 // b) Create the function that makes the test pass.
+// Use tool new Set to iterate over set then give use value one time without repition.
+// Use .concat() on both arrays to return a shallow copy of the original elements combined from the original arrays.
+
+const allNumsNoRepeats = (arrayOne, arrayTwo) => {
+  let concatArray = arrayOne.concat(arrayTwo);
+  let newArray = [...new Set(concatArray)];
+  return newArray;
+};
+// Green - Passed
```

Comment:
Looks great! You could shorten your code slightly by just returning the new set rather than creating a variable just to return it. `return [...new Set(concatArray)];`